standard/logic.lua

```diff
@@ -13,7 +13,7 @@ local Logic = {}
 ---@param val1 A?
 ---@param val2 B?
 ---@param default C?
----@return A|B|C|nil
+---@return A|B|C
```

Comment:
Make sure this actually warns on one input being `nil`, I think that was the problem without it

Reply:
the problem is currently it warns even if none of the inputs is empty